.github/workflows/release.yml

```diff
@@ -1,5 +1,6 @@
 name: Release
 
+###_BEGIN_TASKCTL_SKIP_###
```

Comment:
Does this still do anything?

Reply:
not right now - probably will never be implemented, but would rather delete it all as part of the v2 stabilisation